client/src/components/Q&A/QuestionEntry.jsx

```diff
@@ -1,19 +1,30 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 import AnswerEntry from './AnswerEntry.jsx';
+import AnswerModal from './AnswerModal.jsx';
 import styling from './questions.css';
 
 const QuestionEntry = (props) => {
   const [answerCount, setCount] = useState(2);
   const [answers, setAnswers] = useState([]);
   const [extra, setExtra] = useState(false);
+  const [helpfulnessClick, setHelpfulnessClick] = useState(false);
+  const [answerModal, setAnswerModal] = useState(false);
+
+  // Destructuring of props
+  const { id, helpfulness } = props;
 
   const getAnswers = () => {
-    axios.get(`/qa/questions/${props.id}/answers`)
+    axios.get(`/qa/questions/${id}/answers`)
       .then((response) => {
         const { results } = response.data;
-        results.sort((a, b) => b.helpfulness - a.helpfulness);
-        setAnswers(results);
+        // Sorts answers whether the user is a 'seller' then by helpfulness
+        const sellerAnswers = results.filter(answer => answer.answerer_name.toLowerCase() === 'seller');
```

Comment:
@afong888 When I opened this file in vscode, eslint had underlined this line to indicate that (answer) should be in parentheses. For consistency, can you make the eslint updates recommended? You can right click and select "fix all" and these should be fixed automatically. We can probably ignore the props validation error messages though.